### PR TITLE
Dev: Elastic Search & Grafana setup dev with pino logger

### PR DESCRIPTION
**Pull request by ashishguptasanu**
## Summary

- Setup elasticsearch & grafana as optional feature, while setting up xrengine on local.
- Connect pino logger with elastic search to send logs to elasticsearch & grafana.
- Dev setup docs updated

## References

Partially closes for dev #5885 

## Checklist
- [x] CI/CD checks pass `npm run check`
  - [x] Linter passing via `npm run lint`
  - [x] Typescript passing via `npm run check-errors`
  - [x] Unit & Integration tests passing via `npm run test`
  - [x] Docker build process passing via `npm run build-client`
- [x] If this PR is still a WIP, convert to a draft 
- [x] When this PR is ready, mark it as "Ready for review"
- [x] Changes have been manually QA'd
- [x] Changes reviewed by at least 2 approved reviewers

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

## Reviewers

_Reviewers for this PR_

### CODE REVIEW

File: .env.local.default [hunk 1]
```diff
@@ -200,4 +200,8 @@ DEFAULT_USER_SCOPES=editor:write,location:read,location:write
 BLOCKCHAIN_URL=http://localhost:8080/api/v1
 BLOCKCHAIN_URL_SECRET=secret
 
-FRONTEND_SERVICE_URL=https://local-matchmaking.xrengine.io/v1/frontendservice
\ No newline at end of file
+FRONTEND_SERVICE_URL=https://local-matchmaking.xrengine.io/v1/frontendservice
+
+#define logging url
+ELASTIC_HOST_DEV=http://localhost:9200
```
Comment (dmitrizagidulin) [inline .env.local.default:206]: Let's call this `ELASTIC_HOST_LOCAL` -- dev is actually a separate cluster located at https://dev.theoverlay.io/

Comment (HexaField) [reply .env.local.default:206]: we should only need one env var `ELASTIC_HOST` since these contexts are all configured separately

**Comment by dmitrizagidulin** [comment]
@ashishguptasanu - excellent, ok!
I checked out this branch, installed, and was able to pull up the Grafana UI:

Screen Shot 2022-05-03 at 7 21 01 PM

How do I view the logs via this UI?

**Comment by ashishguptasanu** [comment]
> @ashishguptasanu - excellent, ok! I checked out this branch, installed, and was able to pull up the Grafana UI:
> 
> Screen Shot 2022-05-03 at 7 21 01 PM
> 
> How do I view the logs via this UI?

Go to settings > Index Patterns > Create Index Patterns > Type name: xr-engine* > Next > Select time as time filter > Create Index pattern
Then go to Discover & check index xr-engine

**Review by dmitrizagidulin — APPROVED** [review]
:+1: 
@ashishguptasanu - it looks like you got Pino 7 working, with the multiStream transport?? Excellent.

**Comment by dmitrizagidulin** [comment]
(Tested this PR locally, can confirm it works.)

**Review by barankyle — APPROVED** [review]

**Pull request merged**